account/routes.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+const express = require('express');
+const {requiresAuth} = require('express-openid-connect');
+const createTemplateEngineService = require('../templateEngine');
+const getValidReferrerOrDefault = require('./utils/getValidReferrerOrDefault');
+const {getDashboardURI} = require('./utils/getActionURIs');
+
+const router = express.Router();
+
+router.get('/sign-in/success', requiresAuth(), (req, res, next) => {
+    try {
+        const expiryDate = new Date(
+            new Date().setDate(new Date().getDate() + 31)
```

Comment:
Check this is the correct day to show

Reply:
needs discussion around how this is derived. CW? DCS /eta endpoint?

Reply:
needs discussion around how this is derived. CW? DCS /eta endpoint?